op-e2e/config/init.go

```diff
@@ -291,8 +291,9 @@ func initAllocType(root string, allocType AllocType) {
 				"l2GenesisHoloceneTimeOffset": nil,
 				"l2GenesisIsthmusTimeOffset":  nil,
 				// SWC changes
-				"useSoulGasToken":      true,
-				"isSoulBackedByNative": true,
+				"useSoulGasToken":         true,
+				"isSoulBackedByNative":    true,
+				"l2GenesisBlobTimeOffset": "0x0",
```

Comment:
And looks like this change is also unnecessary since it's handled in the test [here](https://github.com/QuarkChain/optimism/blob/51f8f51816eb61b1ae4a9c3eab7333d7d2ea6b27/op-e2e/l2blob/l2blob_test.go#L86)

Reply:
Still [fails](https://app.circleci.com/pipelines/circleci/9n5VnJ75qFpuj33eU2XAuA/UykduNDM3FCwRU5gTixTkY/23/workflows/dfbbfd22-250d-4e7e-9b7e-f9f1bfeecf5f/jobs/403/tests) without  `useSoulGasToken` set

Reply:
I mean `l2GenesisBlobTimeOffset` is not needed, this is what this PR is change, right?

Reply:
Sorry I meant `l2GenesisBlobTimeOffset`

Reply:
Looks like it's failing because we changed `require.ErrorContains` to `require.NoError` in this PR: https://github.com/ethstorage/optimism/pull/140 . Maybe we can revert this change since l2 blob is not enabled by default and only enabled in the l2 blob e2e test. This way we don't need to touch the tests of upstream.